app/victoria-metrics/main_test.go

```diff
@@ -77,28 +71,32 @@ type Metric struct {
 	Timestamps []int64           `json:"timestamps"`
 }
 
+func (r *Metric) UnmarshalJSON(b []byte) error {
+	type plain Metric
+	return json.Unmarshal(testutil.PopulateTimeTpl(b, insertionTime), (*plain)(r))
+}
+
 type Series struct {
 	Status string              `json:"status"`
 	Data   []map[string]string `json:"data"`
 }
+type Query struct {
+	Status string    `json:"status"`
+	Data   QueryData `json:"data"`
+}
+type QueryData struct {
+	ResultType string            `json:"resultType"`
```

Comment:
inconsistent name - camelCase vs snake_case

Reply:
https://prometheus.io/docs/prometheus/latest/querying/api/
the schema comes from Prom response, can't control it